lib/livebook_cli/server.ex

```diff
@@ -252,6 +268,15 @@ defmodule LivebookCLI.Server do
     |> URI.to_string()
   end
 
+  defp update_query(url, params) do
```

Comment:
We have `Livebook.Utils.append_query`, you can change it `def` and reuse it here :)